.vscode/settings.json

```diff
@@ -136,6 +136,6 @@
   },
   "editor.formatOnSave": true,
   "[typescript]": {
-    "editor.defaultFormatter": "biomejs.biome"
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
I believe we want to keep Biome here, unless the team agrees to switch to Prettier 😋